src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -274,36 +230,30 @@ class BankAccountStep extends React.Component {
                     </ScrollView>
                 )}
                 {subStep === CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID && (
-                    <FormScrollView>
-                        <View style={[styles.mh5, styles.mb5, styles.flex1]}>
-                            <AddPlaidBankAccount
-                                text={this.props.translate('bankAccount.plaidBodyCopy')}
-                                onSelect={(params) => {
-                                    this.setState({
-                                        selectedPlaidBankAccount: params.selectedPlaidBankAccount,
-                                    });
-                                }}
-                                onExitPlaid={() => BankAccounts.setBankAccountSubStep(null)}
-                                receivedRedirectURI={this.props.receivedRedirectURI}
-                                plaidLinkOAuthToken={this.props.plaidLinkOAuthToken}
-                                allowDebit
-                            />
-                        </View>
-                        {!_.isUndefined(this.state.selectedPlaidBankAccount) && (
```

Comment:
I think we should keep this check in the refactored form. As removing it will keep the submit button displayed even when Plaid is still loading or no account has been selected yet. We could add to `Form` an optional prop `isSubmitButtonVisible` and pass this check to it. @luacmartins what do you think?

Reply:
The suggestion sounds good, @luacmartins should we add a new prop `isSubmitButtonVisible`for hiding the form button?

Reply:
Sounds good to me 👍